app/models/trend.rb

```diff
@@ -6,8 +6,10 @@ class Trend < ApplicationRecord
   belongs_to :trendable, :polymorphic => true
   has_many :trend_counters, :dependent => :destroy
 
+  scoped_search :on => [:trendable_type]
```

Comment:
:trendable_type does not have to be specified as an array if that's single value